qa/vagrant/src/test/resources/packaging/utils/utils.bash

```diff
@@ -415,6 +415,50 @@ stop_elasticsearch_service() {
     fi
 }
 
+# the default netcat packages in the distributions we test are not all compatible
+test_port() {
+    local host="$1"
+    local port="$2"
+    cat < /dev/null > "/dev/tcp/$host/$port"
+}
+
+describe_port() {
+    local host="$1"
+    local port="$2"
+    run test_port "$host" "$port"
```

Comment:
I'd put this in a local variable, I like the utilities not depend on the `run` wrapper provided by the bats framework..; wdyt?